docs/content/docs/dev/table/sql/describe.md

```diff
@@ -26,7 +26,7 @@ under the License.
 
 # DESCRIBE Statements
 
-DESCRIBE statements are used to describe the schema of a table or a view, or the metadata of a catalog, or the specified job in the Flink cluster.
+DESCRIBE statements are used to describe the schema of a table or a view, or the metadata of a catalog or a function, or the specified job in the Flink cluster.
```

Comment:
Can you add the English to https://github.com/apache/flink/blob/master/docs/content.zh/docs/dev/table/sql/describe.md as well?

Reply:
added